packages/commons/ir-utils/src/examples/injectAutogeneratedExamples.ts

```diff
@@ -0,0 +1,41 @@
+import { IntermediateRepresentation } from "@fern-api/ir-sdk";
+import { generateEndpointExample } from "./generator/generateSuccessEndpointExample";
+
+export interface ExampleGenerationArgs {
+    disabled: boolean;
+    includeOptionalRequestPropertyExamples?: boolean;
+    skipAutogenerationIfManualExamplesExist?: boolean;
+}
+
+export function injectAutogeneratedExamples({
+    ir,
+    exampleGeneration
+}: {
+    ir: Omit<IntermediateRepresentation, "sdkConfig" | "subpackages" | "rootPackage">;
+    exampleGeneration: ExampleGenerationArgs;
+}): Omit<IntermediateRepresentation, "sdkConfig" | "subpackages" | "rootPackage"> {
+    for (const [_, service] of Object.entries(ir.services)) {
+        for (const endpoint of service.endpoints) {
+            if (
+                endpoint.userSpecifiedExamples.length > 0 &&
+                exampleGeneration?.skipAutogenerationIfManualExamplesExist === true
```

Comment:
nit: equivalent to 
```suggestion
                exampleGeneration?.skipAutogenerationIfManualExamplesExist
```

Reply:
will update! (note: this was all copy pasting than rewriting)